core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -0,0 +1,128 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the body of the simple editor.
+ * editor.
+ */
+
+oppia.directive('simpleEditorBody', [function() {
+  return {
+    restrict: 'E',
+    scope: {},
+    templateUrl: 'simpleEditor/body',
+    controller: [
+      '$scope', '$rootScope', '$timeout', 'SimpleEditorQuestionsDataService',
+      'explorationTitleService', 'explorationInitStateNameService',
+      'explorationStatesService',
+      function(
+          $scope, $rootScope, $timeout, SimpleEditorQuestionsDataService,
+          explorationTitleService, explorationInitStateNameService,
+          explorationStatesService) {
+        $scope.explorationTitleService = explorationTitleService;
+        $scope.questions = [];
+
+        $scope.$on('simpleEditorLoaded', function() {
+          $scope.questions = (
+              SimpleEditorQuestionsDataService.getQuestionsInOrder());
+          $scope.initialState = explorationStatesService.getState(
+            explorationInitStateNameService.savedMemento);
+        });
+
+        $scope.saveTitle = function(newTitle) {
+          explorationTitleService.displayed = newTitle;
+          explorationTitleService.saveDisplayedValue();
+        };
+        $scope.saveIntroduction = function(newHtml) {
+          var initStateName = explorationInitStateNameService.savedMemento;
+          explorationStatesService.saveStateContent(initStateName, [{
+            type: 'text',
+            value: newHtml
+          }]);
+          // We reload the initialState because a fresh copy of the state is
+          // created in the states dict on every update.
+          $scope.initialState = explorationStatesService.getState(
+            explorationInitStateNameService.savedMemento);
+        };
+
+        $scope.saveCustomizationArgs = function(
+            stateName, newCustomizationArgs) {
+          explorationStatesService.saveInteractionCustomizationArgs(
+            stateName, newCustomizationArgs);
+        };
+        $scope.saveAnswerGroups = function(stateName, newAnswerGroups) {
+          explorationStatesService.saveInteractionAnswerGroups(
+            stateName, newAnswerGroups);
+        };
+        $scope.saveDefaultOutcome = function(stateName, newDefaultOutcome) {
+          explorationStatesService.saveInteractionDefaultOutcome(
+            stateName, newDefaultOutcome);
+        };
+        $scope.saveBridgeText = function(stateName, newHtml) {
```

Comment:
Because it provides a smooth transition from one question to the next (this is what we came up with in the UI design process). There are some open questions here, though, so this may change, but I need to talk with the design team first.